src/PresentationalComponents/ReportViewPage/__snapshots__/ReportViewPage.test.tsx.snap

```diff
@@ -87,7 +87,7 @@ exports[`ReportViewPage expect to render tabs when fetch status is complete 1`]
     <PageHeaderTitle
       title={
         <React.Fragment>
-          <Breadcrumb
+          <Unknown
```

Comment:
I wonder what's causing these snapshots to render "Unknown" instead of some of the actual component names. Definitely not a big deal or a blocker, but I'm curious if it's something we could fix here or if it would need to be fixed in patternfly-react.

Reply:
@mturley Thank you very much for spending some time reviewing this PR, I really appreciate it :)... regarding this `<Unknown>` tag, it seems to me that it comes from patternfly-react, take a look to this link for instance: https://github.com/patternfly/patternfly-react/blob/6a11090171c69209a771b3ef1685377dc0f3aa71/packages/patternfly-4/react-core/src/components/OptionsMenu/__snapshots__/OptionsMenu.test.js.snap#L37 and the same for the `<Component` tag https://github.com/patternfly/patternfly-react/blob/6a11090171c69209a771b3ef1685377dc0f3aa71/packages/patternfly-4/react-core/src/layouts/Toolbar/__snapshots__/Toolbar.test.tsx.snap#L4 do you know if it the expected behaviour or should we create a new issue for that?

Reply:
Let's create an issue on patternfly-react for that. I think it's very low priority but snapshots should reflect the real display name of a component. It looks to me like since `OptionsMenu` now renders a `Context.Provider` at the top level, enzyme can't figure out what the actual component is for some reason. My guess is that setting a `displayName` property on the component before exporting it would fix this.

Reply:
Looks like there are others with this issue, it was brought up on PatternFly slack. I'll chime in there.

Reply:
Issue is open now: https://github.com/patternfly/patternfly-react/issues/3317 and I think I have a fix for it